brainglobe_atlasapi/atlas_generation/atlas_scripts/kim_devccf_mouse.py

```diff
@@ -0,0 +1,242 @@
+import json
+import time
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pooch
+from brainglobe_utils.IO.image import load_nii
+from rich.progress import track
+
+from brainglobe_atlasapi import utils
+from brainglobe_atlasapi.atlas_generation.mesh_utils import (
+    Region,
+    create_region_mesh,
+)
+from brainglobe_atlasapi.atlas_generation.wrapup import wrapup_atlas_from_data
+from brainglobe_atlasapi.config import DEFAULT_WORKDIR
+from brainglobe_atlasapi.structure_tree_util import get_structures_tree
+
+ATLAS_NAME = "kim_devccf_mouse"  # multiple versions of the same atlas
+SPECIES = "Mus musculus"
+ATLAS_LINK = "https://kimlab.io/brain-map/DevCCF/"
+CITATION = "Kronman, F.N., Liwang, J.K., Betty, R. et al. 2024, https://doi.org/10.1038/s41467-024-53254-w"
+ORIENTATION = ["left", "superior", "posterior"]
+ROOT_ID = 15564
+RESOLUTION_UM = 20
+VERSION = 1
+PACKAGER = "Carlo Castoldi <castoldi[at]ipmc.cnrs.fr>"
+ATLAS_FILE_URL = "https://doi.org/10.6084/m9.figshare.26377171.v1"
+
+TIMEPOINTS = ("E11.5", "E13.5", "E15.5", "E18.5", "P04", "P14", "P56")
+RESOLUTIONS = (20, 50)
+MODALITIES = (
+    "LSFM",  # Light Sheet Fluorescence Microscopy
+    "MRI-adc",  # MRI Apparent Diffusion Coefficient
+    "MRI-dwi",  # MRI Difusion Weighted Imaging
+    "MRI-fa",  # MRI Fractional Anisotropy
+    "MRI-MTR",  # MRI Magnetization Transfer Ratio
+    "MRI-T2",  # MRI T2-weighted
+)
```

Comment:
These are never used AFAICT - what is the intention for them? IIUC they are possible other modalities and resolutions that we don't loop over in the current version... maybe we want them in a later version?

Reply:
Is there any reason why we can't just add them? We have all the modalities for the P56 atlas already.

Reply:
Happy to add them, no reason why not - my question was more about what the intention of this PR is, because the current code puts the modalities into a list variable that is never used?

Reply:
The existence of this list was intended as ~metadata of Kim's DevCCFv1 reference images, rather than for actual generation of the atlases.

The intent of this PR was to add LSFM only because:
* its the most accurate and at highest resolution (20um vs 50 um of MRI images)
* generating 7×6=42 atlases seemed too overwhelming for an user navigating the atlases through `show_atlases()` (et simil.) output. The atlases are 7, the only thing changing is the reference image. This, in my experience, creates a sort of "bias" and confusions.

That said I think i could do adapt the script to generate those atlases as well, if you confirm me you think it's appropriate! However, i would discourage from officially adding them to BrainGlobe, until #141 is not complete.

Reply:
I think it's appropriate. Ideally we would wait until #141 is complete, but unfortunately, if we treat #141 as a blocker, there's a lot of things we won't get done!

Reply:
as you wish. I modified the script to allow other modalities as well.

Now the last thing remaining is the naming convention of the atlases. What do you prefer?
1. `kim_devccf_mouse_<MODALITY>_<AGE>`
2. `kim_devccf_mouse_<AGE>_<MODALITY>`

The 2nd option is probably  better for navigation